### PR TITLE
(#56) Update generic - component usage examples

### PR DESCRIPTION
**Pull request by tdiesler**

### CODE REVIEW

**Comment by tdiesler** [comment]
Ready to merge

File: generic-examples/http/files/public.cert [hunk 1]
```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
```
Comment (squakez) [inline generic-examples/http/files/public.cert:1]: No, we should never put any keystore or certificates even if its a testing one. We can attract the interest of hacking maneuvers. Not sure if it was on purpose or they slipped into the commit by mistake.

**Review by squakez — CHANGES_REQUESTED** [review]
Please, remove the certificates for security reasons.

File: generic-examples/http/files/public.cert (hunk 1 above)
Comment (tdiesler) [reply generic-examples/http/files/public.cert]: Done

**Review by squakez — APPROVED** [review]

**Pull request merged**